onnxruntime/test/providers/nv_tensorrt_rtx/nv_basic_test.cc

```diff
@@ -24,6 +25,35 @@ namespace onnxruntime {
 
 namespace test {
 
+template <typename T>
+class NvExecutionProviderTest : public ::testing::Test {
+ protected:
+  std::string getTypeAsName() {
+    std::string dtype_name = "";
+    if constexpr (std::is_same<T, double>::value) {
+      dtype_name = "fp64";
+    } else if constexpr (std::is_same<T, float>::value) {
+      dtype_name = "fp32";
+    } else if constexpr (std::is_same<T, BFloat16>::value) {
+      dtype_name = "fp16";
+    } else if constexpr (std::is_same<T, MLFloat16>::value) {
+      dtype_name = "bf16";
```

Comment:
Was this missed in the original review, it seems we are setting the BFloat16 type to fp16 rather than bf16 and vice-versa?